src/ai/index.ts

```diff
@@ -0,0 +1,260 @@
+import { CrowdinApi, PaginationOptions, PatchRequest, ResponseList, ResponseObject } from '../core';
+
+export class AI extends CrowdinApi {
+    /**
+     * @param options optional parameters for the request
+     * @see https://developer.crowdin.com/enterprise/api/v2/#operation/api.ai.prompts.getMany
+     */
+    listAIPrompts(options?: AIModel.ListAIPromptsOptions): Promise<ResponseList<AIModel.AIPromptResponse>> {
```

Comment:
We are already in the context of AI API resource. I think we can omit this `AI` part in every method. It will improve readability. @yevheniyJ what do you think?

Reply:
For consistency let's keep `AI` prefix. As this is how we have everywhere else. We might revisit this naming convention. But definitely not in scope of this PR